lib/command/CommandClient.js

```diff
@@ -210,7 +210,8 @@ class CommandClient extends Client {
         }
     }
 
-    async onMessageReactionEvent(msg, emoji, userID) {
+    async onMessageReactionEvent(msg, emoji, reactor) {
+        const userID = reactor.id || reactor;
```

Comment:
the types indicate that `reactor` will be either `Member` or `{ id: string }`. is the `|| reactor` necessary?

Reply:
Yes, reaction remove only passes the user ID

Reply:
Cool, then the types should be adjusted to reflect that :^)

Reply:
`reactor` in `messageReactionAdd` is `Member | { id: string })`

but here `reactor` in `onMessageReactionEvent` is `Member | { id: string }) | string`

Reply:
Yeah and `messageReactionRemove` is `string`, so what's currently there is correct